gcloud/bigtable/happybase/__init__.py

```diff
@@ -16,4 +16,136 @@
+  add a ``cluster`` keyword argument to allow user's to pass in their own
+  :class:`.Cluster` (which they can construct beforehand).
+
+  For example:
+
+  .. code:: python
+
+      from gcloud.bigtable.client import Client
+      client = Client(project=PROJECT_ID, admin=True)
+      cluster = client.cluster(zone, cluster_id)
+      cluster.reload()
+
+      from gcloud.bigtable.happybase import Connection
+      connection = Connection(cluster=cluster)
+
+* Any uses of the ``wal`` (Write Ahead Log) argument will result in a
+  warning as well. This includes uses in:
+
+  * :class:`.Batch` constructor
+  * :meth:`.Batch.put`
+  * :meth:`.Batch.delete`
+  * :meth:`Table.put() <gcloud.bigtable.happybase.table.Table.put>`
+  * :meth:`Table.delete() <gcloud.bigtable.happybase.table.Table.delete>`
+  * :meth:`Table.batch() <gcloud.bigtable.happybase.table.Table.batch>` factory
+* When calling :meth:`.Connection.create_table`, the majority of HBase column
+  family options cannot be used. Among
+
+  * ``max_versions``
+  * ``compression``
+  * ``in_memory``
+  * ``bloom_filter_type``
+  * ``bloom_filter_vector_size``
+  * ``bloom_filter_nb_hashes``
+  * ``block_cache_enabled``
+  * ``time_to_live``
+
+  Only ``max_versions`` and ``time_to_live`` are availabe in Cloud Bigtable
+  (as
+   :class:`MaxVersionsGCRule <gcloud.bigtable.column_family.MaxVersionsGCRule>`
+   and
+   `MaxAgeGCRule <gcloud.bigtable.column_family.MaxAgeGCRule>`).
+
+  In addition to using a dictionary for specifying column family options,
+  we also accept instances of :class:`.GarbageCollectionRule` or subclasses.
+* :meth:`Table.scan() <gcloud.bigtable.happybase.table.Table.scan>` no longer
+  accepts the following arguments (which will result in a warning):
+
+  * ``batch_size``
+  * ``scan_batching``
+  * ``sorted_columns``
+
+* Using a HBase filter string in
+  :meth:`Table.scan() <gcloud.bigtable.happybase.table.Table.scan>` is
+  not possible with Cloud Bigtable and will result in a
+  :class:`TypeError <exceptions.TypeError>`. However, the method now accepts
+  instances of :class:`.RowFilter` and subclasses.
+* :meth:`.Batch.delete` (and hence
+  :meth:`Table.delete() <gcloud.bigtable.happybase.table.Table.delete>`)
+  will fail with a :class:`ValueError <exceptions.ValueError>` when either a
+  row or column family delete is attempted with a ``timestamp``. This is
+  because the Cloud Bigtable API uses the ``DeleteFromFamily`` and
+  ``DeleteFromRow`` mutations for these deletes, and neither of these
+  mutations support a timestamp.
 """
+
+from gcloud.bigtable.happybase.batch import Batch
+from gcloud.bigtable.happybase.connection import Connection
+from gcloud.bigtable.happybase.connection import DEFAULT_HOST
+from gcloud.bigtable.happybase.connection import DEFAULT_PORT
+from gcloud.bigtable.happybase.pool import ConnectionPool
+from gcloud.bigtable.happybase.pool import NoConnectionsAvailable
+from gcloud.bigtable.happybase.table import Table
```

Comment:
@jonparrott [Here][1] is `happybase.__init__` for reference.

[1]: https://github.com/wbolster/happybase/blob/9cbd718c10a3089f234f1eac1236b631e1f8e7cd/happybase/__init__.py